draft.md

```diff
@@ -247,6 +248,12 @@ You can visit https://www.pling.com/product/add to register a new product on the
 
 The __bintray-zsync__ transport was there to support [Bintray](https://bintray.com/). It is deprecated.
 
+### Runtime
+
+The AppImage runtime is the program that mounts the filesystem image and executes the payload.
+
+If the AppImage runtime is a static binary, then it **MUST** support the environment variable `TARGET_APPIMAGE`. If set to an existing path, then the AppImage Runtime will use the filesystem image from the AppImage at that path for all its operations rather than the filesystem image appended to the runtime binary.
```

Comment:
```suggestion
If the AppImage runtime is a static binary, then it **MUST** support the environment variable `TARGET_APPIMAGE`. All runtime files **SHOULD** implement this parameter, too. If set to an existing path, then the AppImage Runtime will use the filesystem image from the AppImage at that path for all its operations rather than the filesystem image appended to the runtime binary.
```